projects/plugins/jetpack/modules/firewall.php

```diff
@@ -0,0 +1,90 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Module Name: Firewall
```

Comment:
Do we want to be naming things as "WAF" and not "Firewall" everywhere in the codebase? I chose "Firewall" here as that is the actual title of the module that the user interacts with.

Reply:
Wouldn't it be easier if we called it "WAF" everywhere? What would be the distinction between when we call it WAF and when we call it firewall? At least on the technical side, it has been WAF everywhere that I am aware of so far.